index.htm

```diff
@@ -0,0 +1,16 @@
+<head>
+    <title>Dyno Bros</title>
+    <style>
+        * {
+            padding: 0;
+            margin: 0;
+        }
+    </style>
+</head>
+
+<canvas id="game-canvas"></canvas>
+<script src="main.js"></script>
+
+<body>
```

Comment:
this body needs to be at line 10, making canvas inside it

Reply:
yessir